spec/controllers/email_addresses_controller_spec.rb

```diff
@@ -5,11 +5,11 @@
   let(:person) { Person.create(first_name: "Bobby", last_name: "Jones") }
 
   let(:valid_attributes) {
-    { address: 'molsen111@example.com', person_id: person.id }
+    { address: 'molsen111@example.com', contact_id: person.id, contact_type: 'Person' }
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.